app/controllers/participants_controller.rb

```diff
@@ -34,6 +34,18 @@ def lookup
   # GET /participants/1.json
   def show
     @memberships = @participant.memberships.all
+
+    building_statuses = @memberships.map { |m| m.organization.organization_category.building_status }
+    building = building_statuses.include?(true)
+    if @memberships.empty?
+      @wristband = "None - No organizations"
+    elsif !@participant.has_signed_waiver
+      @wristband = "None - No waiver signature"
+    elsif building
+      @wristband = "Red"
+    else
+      @wristband = "Blue"
```

Comment:
Can we please use a reasonable constant here rather than "Red" and "Blue" This new code will already have to change yearly (which sucks), but we should at least have semantically named constants